autoload/neomake.vim

```diff
@@ -531,6 +536,7 @@ function! s:Make(options) abort
         let s:make_id += 1
         let s:make_options[s:make_id] = {
                     \ 'cwd': getcwd(),
+                    \ 'verbosity': get(g:, 'neomake_verbose', 1) + &verbose,
```

Comment:
It's not possible to detect if `:verb` is used, except for assuming that it is when it's `> 0`.
This means that `&verbose` alone cannot be used as the default, since that would be `0` then.
The default is 1 currently, but that's not set in stone, and logging levels should be reviewed anyway - currently I'm just stuffing most of it into the debug level (3).